parsons-tool-frontend/src/pages/Summary.js

```diff
@@ -43,8 +41,7 @@ function Summary() {
             >
               https://docs.google.com/forms/d/e/1FAIpQLSfh2QhZeWG0AgIUqDq2idDSYvOttxHLz8vhIXOqD3EQ5gzONg/viewform?usp=sf_link
             </a>
-            <br />
-            <br /> If you do decide to fill it out, for the <b>Group</b> field, please enter the number <b>"{group}"</b>
+            <br />{' '}
```

Comment:
Why do we need the `{' '}` - can't we just get rid of that completely?

Reply:
I think that was added by format on save